CHANGELOG.md

```diff
@@ -5,12 +5,114 @@ Documentation of all notable changes to the **Fizzy** project.
+
+A notable change is the upgrade from wabt 1.0.12 to 1.0.19 in our testing infrastructure.
+While this allows us to conduct fuzzing, it must be noted there is a [significant speed regression](https://github.com/WebAssembly/wabt/issues/1550)
+with the newer wabt.
+
+There is a slight speed regression compared to Fizzy 0.4.0, which can be attributed to ... what exactly?
+
+#### Instantiation
+
+```
+                         fizzy 0.5.0  fizzy 0.4.0   wabt 1.0.19  wasm3 0.4.7
+                 abs. [µs]  rel. [%]     rel. [%]      rel. [%]     rel. [%]
+geometric mean                   100          101           279           27
+blake2b                 28       100           99           306           38
+ecpairing             1399       100          101           305            3
+keccak256               47       100          103           289           18
+memset                  10       100          100           285           79
+mul256_opt0             12       100          101           246           56
+ramanujan_pi            28       100          101           260           30
+sha1                    43       100          101           284           19
+sha256                  69       100          100           297           12
+taylor_pi                7       100           99           246          116
+```
+
+#### Execution
+
+```
+                               |            fizzy            |        wabt       |  wasm3  |
+                               |       0.5.0       |  0.4.0  |  1.0.12 |  1.0.19 |  0.4.7  |
+                                abs. [µs]  rel. [%]  rel. [%]  rel. [%]  rel. [%]  rel. [%]
+geometirc mean                                  100        95       167       508        26
+blake2b/512_bytes_rounds_1             87       100        97       172       523        26
+blake2b/512_bytes_rounds_16          1329       100        96       172       525        26
+ecpairing/onepoint                 411770       100       102       172       516        38
+keccak256/512_bytes_rounds_1          105       100        96       162       416        18
+keccak256/512_bytes_rounds_16        1548       100        96       162       418        18
+memset/256_bytes                        7       100        96       155       509        28
+memset/60000_bytes                   1623       100        95       155       507        28
+mul256_opt0/input0                     29       100        91       152       545        22
+mul256_opt0/input1                     29       100        89       152       543        22
+ramanujan_pi/33_runs                  138       100        91       163       483        22
+sha1/512_bytes_rounds_1                94       100        97       181       563        29
+sha1/512_bytes_rounds_16             1318       100        96       179       566        29
+sha256/512_bytes_rounds_1              96       100        98       186       539        30
```

Comment:
I guess we should remove the removed tests?

Reply:
They may stay in my opinion.